ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template5.kt

```diff
@@ -0,0 +1,453 @@
+    state: PaywallState.Loaded,
+    packageInfo: TemplateConfiguration.PackageInfo,
+    viewModel: PaywallViewModel,
+) {
+    val colors = state.currentColors
+    val isSelected = packageInfo == state.selectedPackage.value
+
+    val buttonAlpha = viewModel.packageButtonActionInProgressOpacityAnimation()
+    val textColor = colors.text1
+    val borderColor = state.packageButtonColorAnimation(
+        packageInfo = packageInfo,
+        selectedColor = colors.selectedOutline,
+        unselectedColor = colors.unselectedOutline,
+    )
+
+    Button(
+        modifier = Modifier
+            .fillMaxWidth()
+            .alpha(buttonAlpha)
+            .align(Alignment.Start)
+            .semantics {
+                selected = isSelected
+            },
+        onClick = { viewModel.selectPackage(packageInfo) },
+        colors = ButtonDefaults.buttonColors(containerColor = Color.Transparent, contentColor = textColor),
+        shape = RoundedCornerShape(UIConstant.defaultPackageCornerRadius),
+        contentPadding = PaddingValues(
+            vertical = UIConstant.defaultVerticalSpacing,
+            horizontal = UIConstant.defaultHorizontalPadding,
+        ),
+        border = BorderStroke(
+            UIConstant.defaultPackageBorderWidth,
+            borderColor,
+        ),
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth(),
+            verticalArrangement = Arrangement.spacedBy(4.dp),
+            horizontalAlignment = Alignment.Start,
+        ) {
+            Row(horizontalArrangement = Arrangement.spacedBy(6.dp), verticalAlignment = Alignment.CenterVertically) {
+                CheckmarkBox(isSelected = isSelected, colors = colors)
+                Text(
+                    text = packageInfo.localization.offerName ?: packageInfo.rcPackage.product.title,
+                    color = textColor,
+                    style = MaterialTheme.typography.bodyLarge,
+                    fontWeight = FontWeight.SemiBold,
+                )
+                Spacer(modifier = Modifier.weight(1f))
+                DiscountBanner(state = state, packageInfo = packageInfo)
+            }
+            IntroEligibilityStateView(
+                textWithNoIntroOffer = packageInfo.localization.offerDetails,
+                textWithIntroOffer = packageInfo.localization.offerDetailsWithIntroOffer,
+                textWithMultipleIntroOffers = packageInfo.localization.offerDetailsWithMultipleIntroOffers,
+                eligibility = packageInfo.introEligibility,
+                color = textColor,
+                style = MaterialTheme.typography.bodyMedium,
+                allowLinks = false,
+            )
+        }
+    }
+}
+
+@Composable
+private fun CheckmarkBox(isSelected: Boolean, colors: TemplateConfiguration.Colors) {
+    Box(
+        modifier = Modifier
+            .size(Template5UIConstants.checkmarkSize)
+            .clip(CircleShape)
+            // TODO: this is wrong?
```

Comment:
Hmm can't say without debugging and comparing with iOS, but seems that `colors.background` is white, so the check_circle is tinted white, while `colors.selectedOutline` is green (in the screenshot). Maybe we need to use `selectedOutline` as the tint color for the checkmark and have a transparent background if selected?

Reply:
Yes thank you! My Friday brain couldn't figure it out :P

Reply:
<img width="200" alt="Screenshot 2023-10-30 at 10 34 58" src="https://github.com/RevenueCat/purchases-android/assets/685609/f83b18e6-9a1a-4e5c-a6c3-d901fbde6aa8">